src/amo/pages/AddonReviewList/index.js

```diff
@@ -263,6 +257,20 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
         <LoadingText />
       );
 
+    let placeholderCount =
+      addonReviewCount || this.generateRandomLoadingReviews(1, 25);
+    if (placeholderCount > 25) {
+      placeholderCount = 25;
```

Comment:
We should have a constant somewhere (hopefully), that will be better. We use `page_size` in the API if I remember correctly.